src/components/ebay-button/test/test.server.js

```diff
@@ -26,6 +26,18 @@ Object.keys(properties).forEach(property => {
     });
 });
 
+test('renders button with overriden type', context => {
+    const input = { type: 'submit' };
+    const $ = testUtils.getCheerio(context.render(input));
+    expect($(`button.btn`).attr('type')).to.equal('submit');
+});
+
+test('renders button with default type', context => {
```

Comment:
Maybe swap the order of these 2 tests. I.e. first test for default type, then test for overridden type.

Reply:
https://github.com/cestdiego/ebayui-core/commit/3aa5f90